### PR TITLE
fixing spelling errors in comments

### PR DESCRIPTION
**Pull request by tsolodov**
## What was changed
Fixed spelling in code comments

## Why?

Fix spelling errors

## Checklist
N/A

1. Closes 
N/A

2. How was this tested:
N/A

3. Any docs updates needed?
N/A

### CODE REVIEW

File: internal/activity.go [hunk 1]
```diff
@@ -224,7 +224,7 @@ func GetWorkerStopChannel(ctx context.Context) <-chan struct{} {
 //
 //	TODO: we don't have a way to distinguish between the two cases when context is canceled because
 //	context doesn't support overriding value of ctx.Error.
-//	TODO: Implement automatic heartbeating with cancellation through ctx.
```
Comment (Quinn-With-Two-Ns) [inline internal/activity.go:227]: Don't believe this is a spelling mistake

File: internal/client.go [hunk 2]
```diff
@@ -172,7 +172,7 @@ type (
 		GetWorkflowHistory(ctx context.Context, workflowID string, runID string, isLongPoll bool, filterType enumspb.HistoryEventFilterType) HistoryEventIterator
 
 		// CompleteActivity reports activity completed.
-		// activity Execute method can return acitivity.activity.ErrResultPending to
```
Comment (Quinn-With-Two-Ns) [inline internal/client.go:175]: II think the real mistake here is `acitivity.activity` should just be `activity` , it should  probably be `[activity.ErrResultPending]`

Comment (tsolodov) [reply internal/client.go:175]: let me double-check

Comment (tsolodov) [reply internal/client.go:175]: indeed, fixed it

Comment (Quinn-With-Two-Ns) [reply internal/client.go:175]: Thanks!

**Review by Quinn-With-Two-Ns — APPROVED** [review]

**Pull request merged**